src/components/ButtonWithConfirm/ButtonWithConfirm.js

```diff
@@ -95,30 +113,51 @@ class ButtonWithConfirm extends Component<Props, ComponentState> {
     return (
       <div className='button-with-confirm'>
         {isActive ? (
-          <Fragment>
+          <div
+            className={classNames(
+              'actions',
+              color === 'blue' ? '-white' : '-blue',
+              isReverse && '-reverse',
+              isOverlay && '-overlay',
+            )}
+          >
             <JFlatButton
               onClick={this.cancelAction}
               label={labelCancel}
-              color='white'
+              color={color}
               isBordered
             />
             <div className='confirm'>
               <JFlatButton
                 onClick={onClick}
                 label={(countdown > 0) ? `${labelConfirm} – ${countdown} sec` : labelConfirm}
-                color='white'
+                color={color}
                 isDisabled={countdown > 0}
                 isBordered
               />
             </div>
-          </Fragment>
+          </div>
         ) : (
-          <JFlatButton
-            onClick={this.initAction}
-            label={label}
-            color='white'
-            isBordered
-          />
+          <Fragment>
+            {label ? (
+              <JFlatButton
+                onClick={this.initAction}
+                label={label}
+                color={color}
+                isBordered
+              />
+            ) : (
```

Comment:
here should be iconName && !label && (...)

Reply:
also better to rename iconName to tooltipIconName because iconName could be used in JFlatButton, but we use JIcon within JTooltip here

Reply:
the same is about iconColor

Reply:
better to specify that we use these props for tooltip child